src/App.js

```diff
@@ -1,30 +1,79 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Header from 'components/header'
 import { Container } from 'react-bootstrap'
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
 import Home from 'screens/Home'
 import About from 'screens/About'
 import Faq from 'screens/Faq'
 import ContactUs from 'screens/ContactUs'
+import Guide from 'screens/Guide'
 import { GlobalStyle, MainContainer } from 'assets/css/GlobalStyle'
+import ApiCall from 'api'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import { saveResponseData } from 'state/Response'
+
+const App = (props) => {
+  const { ppeGuides, ppeItems } = props
+
+  const getApiResponseData = () => {
+    ApiCall((response) => {
+      const { data } = response
+      props.updateResponseData(data)
+    })
+  }
+
+  if (!ppeGuides) {
+    getApiResponseData()
+  }
```

Comment:
Can you look at the thread in Slack where I describe how to achieve this with Redux Saga and a api client using apisauce?

This pr is getting quite big but I'd like to see the API request and data being stored correctly. Can we focus on that?

Reply:
Redux saga with api call applied you can check now